gatsby/lobid/README.md

```diff
@@ -47,14 +55,20 @@ Make sure `node --version` is at least `v10.18.1` and `npm --version` is at leas
 
    Your site is now running at `http://localhost:8000`!
 
-   *Note*: As the port 8000 is closed on emphytos use the proxy http://gatsbydev.lobid.org/. You'll also see a second link: _`http://localhost:8000/___graphql`_, adjust this to the proxy. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).
-   Also, only the dynamically generated pages (this defined in _src/pages_) are served properly.
+   *Note*: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).
+   Also, only the dynamically generated pages (this defined in _src/pages_) are served properly with `gatsby develop`.
 
    Open the `lobid` directory in your code editor of choice and edit e.g. `src/pages/team-de.js`. Save your changes and the browser will update in real time!
 
+2. **Stage**
 
-2. **Deploy**
+   The following assumes being in the web home on emphytos: `ssh <user>@emphytos`; `sudo su`; `cd /srv/www/htdocs/`.
```

Comment:
Is this really needed, becoming root?

Reply:
As discussed in standup, I verified it works as user `lobid` and updated the README.